### PR TITLE
MVVM with RxSwift

### PR DESCRIPTION
**Pull request by LouiseKlodt**
- first attempt at transforming app to use RxSwift

### CODE REVIEW

File: SoloSet/Controller/ViewController.swift [hunk 1]
```diff
@@ -71,30 +77,43 @@ class ViewController: UIViewController {
...
+                self?.updateViewFromModel(rebuild: true)
+            }).disposed(by: bag)
+        
+        infoButton.rx.tap
+            .subscribe(onNext: { [weak self] in
+                guard let info = self?.infoView else {return}
+                self?.view.addSubview(info)
+            }).disposed(by: bag)
+        
+        let swipe = UISwipeGestureRecognizer()
```
Comment (gringoireDM) [inline SoloSet/Controller/ViewController.swift]: You can do this too with rx. :)

Comment (LouiseKlodt) [reply SoloSet/Controller/ViewController.swift]: I tried installing RxGesture, and also tried specifying the latest version compatible with Swift5 - v3.0.0 - but the project doesn't compile after `pod install`. I'm getting 6 compile errors for RxGesture.

Comment (LouiseKlodt) [reply SoloSet/Controller/ViewController.swift]: https://github.com/RxSwiftCommunity/RxGesture/releases/tag/3.0.0

Comment (LouiseKlodt) [reply SoloSet/Controller/ViewController.swift]: Please ignore. Just resolved the issue with Fabio. Needed to run `pod install --repo-update`.

**Comment by LouiseKlodt** [comment]
moved to other branch

**Pull request closed without merging**